cmd/eno-reconciler/main.go

```diff
@@ -106,10 +102,20 @@ func run() error {
 			remoteConfig.QPS = float32(remoteQPS)
 		}
 	}
-	err = reconciliation.New(recmgr, remoteConfig, discoveryMaxRPS, rediscoverWhenNotFound, readinessPollInterval)
+
+	writeBuffer := flowcontrol.NewResourceSliceWriteBufferForManager(mgr, writeBatchInterval, 1)
```

Comment:
Burst went down from 2 to 1? Should it be adjustable?

Reply:
I think 1 makes more sense - send 1 request immediately when something changes, then wait the debounce interval. So in cases where only a single resource changes we get quick feedback.

Personally I think we're okay hardcoding this for the sake of minimizing flags, but open to adding a flag if you feel strongly.

Reply:
I think it's okay to hardcode it, I was just curious about the change, may be worth adding a comment with the fillowing?
> send 1 request immediately when something changes, then wait the debounce interval. So in cases where only a single resource changes we get quick feedback.

Reply:
Pushed!